core/jazz_ui/src/app/secondary-components/advanced-filters/internal/advanced-filters.component.ts

```diff
@@ -37,6 +35,10 @@ export class AdvancedFiltersComponent implements OnInit {
     periodList: Array<string> = ['15 Minutes','1 Hour','6 Hours','1 Day','7 Days','30 Days'];
     periodSelected:string= this.periodList[0];
 
+    timePeriodList: Array<number> = [1,2,3,4,5,6,7];
```

Comment:
i dont have the context on this

Reply:
As slider had option to select period value from 1-7, kept the same.